modal/sandbox.py

```diff
@@ -388,8 +388,8 @@ async def _get_task_id(self):
         while not self._task_id:
             resp = await self._client.stub.SandboxGetTaskId(api_pb2.SandboxGetTaskIdRequest(sandbox_id=self.object_id))
             self._task_id = resp.task_id
-            # TODO: debug why sending an exec right after a task ID exists fails silently
-            await asyncio.sleep(0.5)
+            if not self._task_id:
```

Comment:
When do we not get a task ID back, and why are we sleeping for those cases? How come we're not removing the sleep completely?

Reply:
We shouldn't hammer the RPC - it would hot loop otherwise.